README.md

```diff
@@ -1002,6 +1002,47 @@ CurrencyFactory.currency(for: .greece)?.code ?? noCurrencyCode
 CurrencyFactory.currency(for: .spain)?.code ?? noCurrencyCode
 CurrencyFactory.currency(for: .unitedStates)?.code ?? noCurrencyCode
 CurrencyFactory.currency(for: .uk)?.code ?? noCurrencyCode
+```
+
+ 🔂 Monostate
+ ------------
+
+ The monostate pattern is another way to achieve singularity. It works through a completely different mechanism, it enforces the behavior of singularity without imposing structural constraints. 
+ So in that case, monostate saves the state as static instead of the entire instance as a singleton.
+ [SINGLETON and MONOSTATE - Robert C. Martin](http://staff.cs.utu.fi/~jounsmed/doos_06/material/SingletonAndMonostate.pdf)
+
+### Example:
+
+```swift
+struct Settings {
+
+    enum Theme {
+        case .old
+        case .new
+    }
+
+    private static var theme: Theme
+
+    var currentTheme: Theme {
+        get { Settings.theme }
+        set(newTheme) { Settings.theme = newTheme }
+    }
+}
+```
+
+### Usage:
+
+```swift
+
+// When change the theme
+let settings = Settings() // Starts using theme .old
+settings.currentTheme = .new // Change theme to .new
+
+//On screen 1
+let screenColor: Color = Settings().currentTheme == .old ? .gray : .white
+
+//On screen 2
+let screenTitle: String = Settings().currentTheme == .old ? "Itunes Connect" : "App Store Connect"
```

Comment:
It's nitpicking, just a side note really, not about the Monostate implementation but this particular line here. 
Should we use current theme to infer the screen title?

Reply:
I agree with you, it is not a better example or approach to change the title, but if my screen, for example, has a static title, maybe I could differ conform theme. I will think of a better example of the monostate.